styles/index.scss

```diff
@@ -73,7 +74,7 @@ section.react-virtualized-pivot-demo{
   .app-menu{
     display: flex;
     flex-wrap: wrap;
-    align-items: center;
+    //align-items: center;
```

Comment:
Alignment looks good without `align-items: center;`. Let's delete the commented out line?